config/dev.exs

```diff
@@ -66,6 +66,10 @@ config :animina, AniminaWeb.Endpoint,
 # Enable dev routes for dashboard and mailbox
 config :animina, dev_routes: true
 
+
+# Configures the llama version
+config :animina, :llama_version, "llama3.1:8b"
```

Comment:
Don't call it `llama_version`. Who says that we are going to use Llama in a year? Maybe we switch to a different LLM. Call it something like `llm_version`.